processor/src/main/java/com/blarley/rxretrobusseed/annotationprocessor/processor/RxRetroBusAnnotationProcessor.java

```diff
@@ -160,12 +169,21 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
                             + baseClassName
                             + ";\n");
 
-            constructorDefinition.append(
-                    "\t\tthis."
-                            + baseClassName
-                            + " = new "
-                            + generatedClass
-                            + "(retrofitBuilder, bus);\n");
+            if (generatedClasses.get(generatedClass)) {
+                constructorDefinition.append(
+                        "\t\tthis."
+                                + baseClassName
+                                + " = new "
```

Comment:
Note that the constructor definitions are now the same except for the last line.. which if we keep to one constructor can just use the ternary operator to append null or retrofitBuilder